build.gradle

```diff
@@ -74,10 +74,20 @@ java {
     withSourcesJar()
 }
 
-configurations.configureEach {
-    // This is a workaround for grails-shell exposing a different version of Groovy
-    // than the one used by Gradle. This causes issues with the Groovy compiler.
-    resolutionStrategy.force "org.codehaus.groovy:groovy-xml:$GroovySystem.version"
+configurations.all {
```

Comment:
Why do you choose `configurations.all` instead of `configurations.configureEach`?

Reply:
It was just copied from another Grails project, I have adjusted it to configurations.configureEach which works well.